src/NeoPixelBus.h

```diff
@@ -84,6 +84,13 @@ template<typename T_COLOR_FEATURE, typename T_METHOD> class NeoPixelBus
     {
     }
 
+    NeoPixelBus(uint16_t countPixels, Stream *s) :
```

Comment:
code style:
asterisk for pointer goes with the type not the variable in a declaration.
Stream* s
variable name is too simple and needs to be more description.
Stream* pixieStream
Stream* stream 
Stream* serial

Reply:
Updated convention and name to pixieStream